tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -385,7 +386,7 @@ final class TestingServer(
     val workspaceCapabilities = new WorkspaceClientCapabilities()
     val textDocumentCapabilities = new TextDocumentClientCapabilities
     textDocumentCapabilities.setFoldingRange(new FoldingRangeCapabilities)
-    val experimental = experimentalCapabilities.getOrElse(
+    val experimental = initializationOptions.getOrElse(
```

Comment:
If you're changing this to `initializationOption`, you should probably change the name and also replace the default with `InitializationOptions.default` rather than `ClientExperimentalCapabilities.Default....`

Reply:
Fixed!